code/src/java/plugin/initiative/gui/Initiative.java

```diff
@@ -2389,7 +2249,7 @@ else if (returnVal == SavingThrowDialog.FAIL_OPTION)
 			combatantUpdated(cbt);
 			State newStatus = cbt.getStatus();
 
-			if (!oldStatus.equals(newStatus) && newStatus == State.Dead)
+			if ((oldStatus != newStatus) && (newStatus == State.Dead))
```

Comment:
I think this is debatable whether this is a best practice to go away from object equality.